include/db/jdb-internal.h

```diff
@@ -113,6 +113,8 @@ gboolean j_db_internal_delete (gchar const* namespace, gchar const* name, bson_t
 gboolean j_db_internal_query (gchar const* namespace, gchar const* name, bson_t const* selector, gpointer* iterator, JBatch* batch, GError** error);
 gboolean j_db_internal_iterate (gpointer iterator, bson_t* metadata, GError** error);
 
+gboolean j_internal_reset (gchar const* namespace, JBatch* batch, GError** error);
```

Comment:
`j_db_internal_reset`?

Reply:
the intention was to have one reset function for all backend types - maybe this should be moved somewhere in the core if it should be available for all backend types